config/core/300-resources/revision.yaml

```diff
@@ -688,9 +688,6 @@ spec:
                   description: DesiredReplicas reflects the desired amount of pods running this revision.
                   type: integer
                   format: int32
-                imageDigest:
```

Comment:
I am wondering what would be the behavior for those old Revisions which have this `imageDigest` presented. I saw the error `admission webhook "webhook.serving.knative.dev" denied the request: mutation failed: cannot decode incoming old object: json: unknown field "imageDigest` in the controller. I suspect it may be related to removing this field.